dali/python/nvidia/dali/experimental/dynamic/_op_builder.py

```diff
@@ -304,9 +376,20 @@ def call(self, *raw_args, batch_size=None, **raw_kwargs):
                 if not is_batch:
                     batch_size = self._max_batch_size or 1
 
+            # Handle rng and _random_state arguments for random operators
+            if has_random_state:
+                _handle_rng_argument(raw_kwargs, is_batch, batch_size)
+
             inputs = []
             kwargs = {}
 
+            def _get_dtype(k):
+                """_random_state is a hidden argument not in the conversion map"""
```

Comment:
If it's not in the conversion map, it's actually better - `dtype` will be `None` and everyone's going to be happy.